packages/api/src/Message.ts

```diff
@@ -55,7 +57,7 @@ export default class Message implements MessageInterface {
       destination,
       ack,
       request_id: requestId,
-    } = JSON.parse(json);
+    } = toSafeNumber(JSONbig.parse(json));
```

Comment:
I added this function because json-bigint has a bug https://github.com/sidorares/json-bigint/issues/63 and I would like to have all numbers <= Number.MAX_SAFE_INTEGER as the standard type. You are not able to use BigNumber directly as React value and it will throw an exception when you try to render it.